engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateBy.java

```diff
@@ -713,11 +732,13 @@ public void processUpdate() {
                             final RowSequence rs = it.getNextRowSequenceThrough(end);
                             builder.appendRowSequenceWithOffset(rs, delta);
                         }
-                        shiftedRows = builder.build();
+                        changedRows = builder.build();
```

Comment:
This currently may include removed rows that overlapped shift ranges. That is totally fine, at least for now while you are inserting `toClear`.